nft/nft.go

```diff
@@ -35,11 +35,11 @@ type NFT struct {
 	hash         NFTHash
 	uri          URI
 	approved     base.Address
-	creators     []RightHolder
-	copyrighters []RightHolder
+	creators     []Signer
```

Comment:
@wyuinche 
The number of signers and creators also needs to be limited.

Reply:
@wyuinche
Is there any validation check for duplicated address in creators and duplicated address in copywrighters?

Reply:
@soonkuk
A limit has been applied to creators and copyrighters. The current limit is 10.

Reply:
> @wyuinche Is there any validation check for duplicated address in creators and duplicated address in copywrighters?

@soonkuk No🥲 I'll address this problem.

Reply:
> > @wyuinche Is there any validation check for duplicated address in creators and duplicated address in copywrighters?
> 
> @soonkuk No🥲 I'll address this problem.

@soonkuk 
Added duplicate detection for signers of 'nft', 'mint-form'.